migrations/20170206142200_setup.js

```diff
@@ -63,15 +71,18 @@ exports.up = function(knex, Promise) {
 			table.integer('method_id').references('keyword.id');
 			table.integer('keyword_id').references('keyword.id');
 			table.integer('evidence_id'); // TODO if we ever have a Locus table, this has a reference to it
+			table.timestamp('created_at').defaultTo(knex.fn.now());
```

Comment:
I changed my mind on this one. Annotation has a created_at date already, so it's redundant to add one on all the annotation subtypes as well.